uaclient/cli.py

```diff
@@ -325,10 +313,30 @@ def action_attach(args, cfg):
 
 
 def get_parser():
+    service_line_tmpl = " - {name}: {description}{url}"
+    description_lines = [__doc__]
+    for name, ent_cls in entitlements.ENTITLEMENT_CLASS_BY_NAME.items():
+        if ent_cls.help_doc_url:
+            url = " ({})".format(ent_cls.help_doc_url)
+        else:
+            url = ""
+        service_line = service_line_tmpl.format(
+            name=name, description=ent_cls.description, url=url
+        )
+        if len(service_line) <= 80:
+            description_lines.append(service_line)
+        else:
+            wrapped_words = []
+            line = service_line
+            while len(line) > 80:
+                [line, wrapped_word] = line.rsplit(" ", 1)
+                wrapped_words.append(wrapped_word)
+            description_lines.extend([line, "   " + " ".join(wrapped_words)])
```

Comment:
This looks to me like it will reverse the order of the wrapped_words. Can we add tests to confirm this is correct, please?